### PR TITLE
Biomed 1189 - medication NER

### PR DESCRIPTION
**Pull request by anton-vasin-qs**

### CODE REVIEW

**Comment by anton-vasin-qs** [comment]
Please don't merge this PR. I found a bug and I am fixing it.

**Comment by anton-vasin-qs** [comment]
@RichenAtMedal Ok. Fixed. Please review these changes.

File: neuroner/brat_to_conll.py [hunk 1]
```diff
@@ -68,41 +73,53 @@ def get_sentences_and_tokens_from_stanford(text, core_nlp):
...
+def get_entities_from_brat(text_filepath, annotation_filepath, verbose=False, model_type: str = None):
     # load text
-    with codecs.open(text_filepath, 'r', 'UTF-8') as f:
-        text =f.read()
+    text = read_text(text_filepath)
+    source = read_text(annotation_filepath)
+    # preprocess Brat lines to remove potential Brat parsing errors related to tabs series
+    source = re.sub('\t{2,}', '\t', source)
+
     if verbose: print("\ntext:\n{0}\n".format(text))
```
Comment (AdamBarryMedal) [inline neuroner/brat_to_conll.py]: Will you replace **all** print statements in production code with calls to our operations logger? `from medal.common.log import operations_logger` Also, we cannot print or log any information that is contained within the documents. If we accidentally happen to print or log text from the documents, our console/logs suddenly become Protected Health Information (PHI) and will cause a huge compliance headache.

File: neuroner/brat_to_conll.py [hunk 2]
```diff
@@ -68,41 +73,53 @@ def get_sentences_and_tokens_from_stanford(text, core_nlp):
...
+            if utils_nlp.replace_unicode_whitespaces_with_ascii_whitespace(text[entity['start']:entity['end']]) != \
                     utils_nlp.replace_unicode_whitespaces_with_ascii_whitespace(entity['text']):
-                    print("Warning: brat text and annotation do not match.")
-                    print("\ttext: {0}".format(text[entity['start']:entity['end']]))
-                    print("\tanno: {0}".format(entity['text']))
-                # add to entitys data
-                entities.append(entity)
+                print("Warning: brat text and annotation do not match.")
+                print("\ttext: {0}".format(text[entity['start']:entity['end']]))
+                print("\tanno: {0}".format(entity['text']))
```
Comment (AdamBarryMedal) [inline neuroner/brat_to_conll.py]: These statements as well. If you really need to print out text from the documents, please use `operations_loggeer.debug()`

Comment (anton-vasin-qs) [reply neuroner/brat_to_conll.py]: @AdamBarryMedal Ok. Done. I changed all `print` methods to Medal's logger. And also I did some refactoring.

**Pull request merged**